DepositUI/Pages/Deposit.razor

```diff
@@ -42,6 +42,58 @@
         </div>
     </div>
 }
+else if (mode == "History")
+{
+<div class="skrollable history">
+    <div class="row">
+        @for (int i = 0; i < 4; i++)
+        {
+            <div class="col-3">
+                @if (deposits != null)
+                {
+                    @for (int j = 0; j < deposits.Count; j++)
+                    {
+                        if (j * 4 + i < deposits.Count)
+                        {
+                            var depositNumber = j * 4 + i;
+                            <div class="deposit-block">
+                                <div class="deposit-details">
+                                    <div class="align-left">
+                                        <span>Percent:</span><br />
+                                        <span>Term:</span><br />
+                                        <span>Sum:</span><br />
+                                        <span>Date:</span>
+                                    </div>
+                                    <div class=" align-right">
+                                        <span>@(deposits[depositNumber].Percent)%</span><br />
+                                        <span>@(deposits[depositNumber].Term)</span><br />
+                                        <span>@(deposits[depositNumber].Amount)$</span><br />
+                                        <span>@(deposits[depositNumber].Date.ToString("dd.MM.yyyy"))</span>
+                                    </div>
+                                </div>
+                                <div class="deposit-btn-cont">
+                                    <button @onclick="() => Details(deposits[depositNumber].Id)" class="details-btn">See details</button>
+                                </div>
+                            </div>
+                        }
+                    }
+                }
+            </div>
+        }
+    </div>
+    <button class="details-btn" @onclick="LoadMore">Load More</button>
+</div>
+}
+else if (mode == "Details")
+{
+    <div class="tab">
+        <DepositDetailsTable DepositDetails="depositDetails"> </DepositDetailsTable>
+        <form action="https://localhost:44320/api/depositcalc/csv">
+            <input type="hidden" name="depositId" value="@depositId" />
+            <button type="submit" class="submit-btn">Export to CSV</button>
+        </form>
+    </div>
+}
 
 @code {
```

Comment:
Please split this file to two. The first file shall contain razor file with only HTML code, the second one shall be nested in the first one and contain C# code. This makes the code easier to understand.

Reply:
completed